tests/test_Model.py

```diff
@@ -1050,3 +1050,60 @@ def test_model_solves_multi_trip_instance():
     assert_({2} in visit_sets)
     assert_({3, 4} in visit_sets)
     assert_({5, 6} in visit_sets)
+
+
+def test_model_simultaneous_pickup_delivery_with_reloading():
```

Comment:
This test case is broken (probably a bug with simultaneous PD) but I don't have the time to investigate right now, so I'm keeping this around for later.